analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S5693_cs.html

```diff
@@ -38,6 +38,23 @@ <h2>Sensitive Code Example</h2>
     }
 }
 </pre>
+<p>In <a href="https://docs.microsoft.com/en-us/troubleshoot/aspnet/create-web-config">Web.config</a>:</p>
```

Comment:
Oops I missed this - the two html files are different.

The RSPECs should be updated in their repo https://github.com/SonarSource/rspec and then the rule-api should be invoked to generated the files.

Reply:
Updated from RSPEC